src/domain/space/space.defaults/space.defaults.service.ts

```
@@ -1,156 +1,133 @@
+    const platformTemplatesManager =
+      await this.platformService.getTemplatesManagerOrFail();
+    let templateID = collaborationData.collaborationTemplateID;
+    if (!templateID) {
+      switch (spaceType) {
+        case SpaceType.CHALLENGE:
+        case SpaceType.OPPORTUNITY:
+          const subspaceTemplate =
+            await this.templatesManagerService.getTemplateFromTemplateDefault(
+              platformTemplatesManager.id,
+              TemplateDefaultType.PLATFORM_SUBSPACE
+            );
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)